### PR TITLE
Allow option for hiding version information

### LINKED ISSUE #24: Question: is it possible to not show electron/chrome/node/v8 info
Looks unnecessary for my case and I'd like to hide it.

### PR DESCRIPTION
**Pull request by alexanderturinske**
- Fixes #24
screen shot 2018-06-26 at 9 33 01 pm

### CODE REVIEW

**Comment by rhysd** [comment]
I'm sorry that I failed to catch this PR. Let me review.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "about-window",
-  "version": "1.11.0",
+  "version": "1.12.0",
```
Comment (rhysd) [inline package.json]: Could you revert this change? Let me create another commit to bump version to create a tag commit after merging this PR.

**Comment by rhysd** [comment]
@alexanderturinske  Could you show me the screenshot how it looks like when setting `use_version_info` to `false`?

File: package.json (hunk 1 above)
Comment (alexanderturinske) [reply package.json]: Will do!

Comment (alexanderturinske) [reply package.json]: Done!

**Comment by alexanderturinske** [comment]
@rhysd 
- Screenshot added to the commit.
- Added option to README
- Option was already in .d.ts, but not documented in README that it was

**Comment by rhysd** [comment]
I will publish new version after merging #26

**Comment by bampakoa** [comment]
@rhysd I think that this PR did not make it to the latest release 🤔

**Comment by rhysd** [comment]
@bampakoa Thank you for reporting that. I released 1.12.1. Could you try it?

**Comment by bampakoa** [comment]
@rhysd it works fine in **1.12.1** Thanks a lot!
